### PR TITLE
Improve setup

### PR DESCRIPTION
**Pull request by allewun**
- Improves text formatting
- Improve error message clarity
- Normalizes error sentence casing across all error messages

pr

### CODE REVIEW

File: src/helpers/configuration_prompt_helpers.sh [hunk 1]
```diff
@@ -22,34 +22,38 @@ function ensure_knows_configuration {
...
+  local main_branch_prompt='Please specify the main development branch by name or number'
 
   while [ -z "$main_branch_input" ]; do
     if [ "$(is_main_branch_configured)" = true ]; then
-      echo -n "Please specify the main development branch by name or number (current value: ${MAIN_BRANCH_NAME}): "
+      echo -n "$main_branch_prompt (current value: $(echo_inline_cyan_bold "$MAIN_BRANCH_NAME")): "
     else
-      echo -n "Please specify the main development branch by name or number (current value: None): "
+      echo -n "$main_branch_prompt (current value: None): "
     fi
```
Comment (charlierudolph) [inline src/helpers/configuration_prompt_helpers.sh]: Thoughts on saving a main_branch_current_value (which would be the main branch in cyan or None) to reduce the duplication? Same for perennial branch prompts.

File: features/git-town/configuration/already_set.feature [hunk 2]
```diff
@@ -21,7 +21,7 @@ Feature: listing the configuration
         2: production
         3: qa
 
-      Please specify the main development branch by name or number (current value: main): Please specify a perennial branch by name or number. Leave it blank to finish (current value: qa):
+      Please specify the main development branch by name or number (current value: main): Please specify a perennial branch by name or number. Leave it blank to finish (current value(s): qa):
```
Comment (kevgo) [inline features/git-town/configuration/already_set.feature]: I think you are more popey than the pope here. The previous version was good enough, everybody understands it, it reads easier, is shorter, and has no nested parentheses.

File: src/helpers/configuration_prompt_helpers.sh [hunk 3]
```diff
@@ -22,34 +22,37 @@ function ensure_knows_configuration {
   fi
 
   local main_branch_input
+  local main_branch_current_value='None'
```
Comment (kevgo) [inline src/helpers/configuration_prompt_helpers.sh]: I would lowercase this. `current value: none` reads better than `current value: None`, unless we write Visual Basic  :)

File: src/helpers/configuration_prompt_helpers.sh [hunk 4]
```diff
@@ -58,17 +61,22 @@ function ensure_knows_configuration {
...
 
   local perennial_branches_input=''
+  local perennial_branches_current_values='None'
+  if [ "$(are_perennial_branches_configured)" = true ]; then
+    perennial_branches_current_values=$(echo_inline_cyan_bold "$PERENNIAL_BRANCH_NAMES")
+  fi
 
   while true; do
-    echo -n "Please specify a perennial branch by name or number. Leave it blank to finish (current value: ${PERENNIAL_BRANCH_NAMES}): "
+    echo -n "Please specify a perennial branch by name or number. Leave it blank to finish (current value(s): $perennial_branches_current_values): "
```
Comment (kevgo) [inline src/helpers/configuration_prompt_helpers.sh]: Same here, don't add `(s)` in my opinion.

File: features/git-town/configuration/already_set.feature (hunk 2 above)
Comment (allewun) [inline features/git-town/configuration/already_set.feature]: pope

Comment (kevgo) [inline features/git-town/configuration/already_set.feature]: Hahaha  😄

**Pull request closed without merging**